### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Jannik143**
I finished a basic menu.java

### CODE REVIEW

File: src/menu.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      System.out.println("Which state do you want to load?");
+     
+      }
+     
+    
+    
+    if (text == 3)
+    {
+      
+      System.out.println("Which state do you want to load?");
```
Comment (suppergerrie2) [inline src/menu.java]: This should exit the program instead of doing the same thing as Load.

**Comment by EhWhoAmI** [comment]
Use switch statements, and not only that, it will fail if you type in characters. also something that is not accepted.

File: src/menu.java (hunk 1 above)
Comment (Jannik143) [reply src/menu.java]: yeah thats true I changed the if's to switch already but what do you mean with enum or constant class?

**Comment by EhWhoAmI** [comment]
even with the changes, it is still not really secure. What if the user enters a number more than 3? What if the user enters letters? It will fail. make a do while loop for the input, and catch exceptions.

**Comment by EhWhoAmI** [comment]
As I said, make a do while loop so that the user can keep typing stuff. No point exiting the program each time the user enters the wrong thing.

**Pull request merged**